src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -140,6 +142,21 @@ public static YouTube getService() throws GeneralSecurityException, IOException
         .build();
   }
 
+  /**
+   * 
+   * Call to add playlist information to datastore.
+   */
+  private void storePlaylist(String playlistItemJson){
+    // Create a game
+    Entity gameEntity = new Entity("Game");
```

Comment:
I feel like this method is doing more than just storing a playlist, it's also creating a game and trying a playlist to that game.

perhaps change the names up to make this clearer?

createGameAndStorePlaylist potentially

Reply:
I think I will just change it to create game entity, because in the future we will want to add more to the entity than just the playlist